init.fish

```diff
@@ -21,6 +21,7 @@ __git_abbr g          git
 __git_abbr ga         git add
 __git_abbr gaa        git add --all
 __git_abbr gapa       git add --patch
+__git_abbr gap        git patch
```

Comment:
My git installs say patch is not a command, and I can't find online docs for this.

Reply:
I just tried it again, it was a mistake on my end. Should by `git apply` instead.